### PR TITLE
updating the option bg color on hover

### PR DESCRIPTION
**Pull request by cuginoAle**
setting a border and BG when hovering the tooltip items
The BG doesn't change when hovering in `presentation mode`

screenshot 2018-11-22 at 12 05 17

### CODE REVIEW

File: visualizer/options-menu.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
   font-size: var(--small-text-size);
   position: absolute;
   z-index: 1;
-  background-color: rgba(var(--opposite-color-val), 0.85);
+  background-color: var(--options-menu-bg-color);
```
Comment (AlanSl) [inline visualizer/options-menu.css]: Can we keep the translucency here: on some profiles the only frames added may be behind the options menu, and we want the user to be able to notice that something has changed.

(I have a plan to move the main filter controls out of the options menu and replace the circles in the key at the bottom with checkboxes, as a better solution to this problem, but that needs to be done after adding the dependency filters)

File: visualizer/style.css [hunk 2]
```diff
@@ -46,7 +46,8 @@ html {
   --footer-bg-color: rgb(15, 18, 26);
   --footer-color: rgba(255, 255, 255, 0.9);
 
-  --options-menu-bg-color: rgba(var(--banner-bg-color-val), 0.9);
+  --options-menu-bg-color: rgb(0, 0, 0);
+  --options-menu-bg-hover: rgb(10, 10, 10);
```
Comment (AlanSl) [inline visualizer/style.css]: For consistency it'd make sense to use this for the tooltip and (non-disabled) button backgrounds on hover as well; and give it a more general name (maybe --clickable-bg-hover).

And then I think we wouldn't need --options-menu-bg-color with the above change

File: visualizer/options-menu.css (hunk 1 above)
Comment (cuginoAle) [reply visualizer/options-menu.css]: Oh, I must have forgot to save the css file.
It looks like this on my machine:
`--options-menu-bg-color: rgba(0, 0, 0, 0.85);`

Anyway, I saw your next comment and will update this accordingly ;)

**Comment by AlanSl** [comment]
LGTM as is.

That the _background color_ specifically has changed doesn't need to be noticeable, it just needs to be intuitively obvious without stopping and thinking that a) that some interactive element is ready to click on and b) which one it is. 

The border already does all of this; so we just want enough consistency that a user won't think "This button is not like the others, what does that mean?" and it's just obvious without stopping to think that this is another button that is just like the other buttons.

**Pull request merged**